libs/client/client.ts

```diff
@@ -38,8 +42,8 @@ class Client {
     logger.debug({ apiUrl }, "new Client at");
     this.apiUrl = apiUrl;
     // Set optional worker addresses if provided
-    this.apiTxUrl = typeof apiTxUrl === 'undefined' ? '' : apiTxUrl;
-    this.apiBpUrl = typeof apiBpUrl === 'undefined' ? '' : apiBpUrl;
+    this.apiTxUrl = typeof apiTxUrl === "undefined" ? "" : apiTxUrl;
```

Comment:
is it possible to force double quotations so that at list its highlighted in an editor as incorrect?

Reply:
Hello @druiz0992 not sure if highlighting is possible. The codebase has a `.prettierrc.yml` file that contains some style rules. It's just a matter of configuring VSCode to pick it up and "correcting" the format for example when saving the files.